search-api/src/search_api/models/user.py

```diff
@@ -51,6 +51,7 @@ class User(db.Model):
     lastname = db.Column(db.String(1000))
     email = db.Column(db.String(1024))
     login_source = db.Column('login_source', db.String(200), nullable=True)
+    idp_userid = db.Column(db.String(256), index=True)
```

Comment:
the current migration doesn't include the index -- can you add that please? Other than that I think this looks good 👍